e2e_playwright/st_heading_test.py

```diff
@@ -22,17 +22,17 @@
 
 def _get_title_elements(app: Page) -> Locator:
     """Title elements are rendered as h1 elements"""
-    return app.locator(".element-container .stMarkdown h1")
+    return app.locator(".element-container [data-testid=stHeading] h1")
```

Comment:
I think it might be better to use `.get_by_test_id("stHeading").locator("h1")` here instead.

Reply:
yep great catch! was a left-over from removing the class name